app/views/posts/index.html.erb

```diff
@@ -1 +1,34 @@
-        <h1> Here is the index page of your posts for a given user</h1>
\ No newline at end of file
+<div class='post-div'>
+    <div class="show-container"> 
+         <img src=<%=@posts.photo%> alt='Profile' class = 'cover-img'  />
+        <div>
+           <p> <%= @posts.name %></h3></p>
+           <p>Number of posts: <%= @posts.posts_counter %> </p>
+        </div>
+    </div>
+
+    <ul>
+        <% @posts.posts.each do |post| %>
+          <li class='post'>
+            <p>Post #<%= post.id%></p>
+            <p>
+                 <%= link_to post.title , "posts/#{post.id}" %>
+            </p>
+            <p><%= post.text %></p>
+            <p>
+                <span>comments: <%= post.comments_counter %></span>, 
+                <span>Likes: <%= post.likes_counter %> </span>
+            </p>
+          </li>
+          <div>
+            <% post.recent_comments.each do |post| %>
+               <p>User : <%= post.author.name %></p>
+               <p><%= post.text %></p>
+             <% end %>
+          </div>
+        <% end %>
+    </ul>
+    <button>
+        <%= link_to 'Pagination' , "/users/#{@posts.id}" %>
+    </button>
+</div>
```

Comment:
- [ ] Just like the previous reviewer opined, kindly implement the designs from the sneak peek wireframes for all your views. You don't need to style it too much but you just need to follow the wireframes as this is a very important requirement🙏 
### Design Guideline
![image](https://github.com/techmoves/movesBloc/assets/46839548/24637ea1-6210-4d11-a4cb-ab4a75f6dedf)

### Your Design
![image](https://github.com/techmoves/movesBloc/assets/46839548/fc31ec8d-9e58-4bd1-a3b5-021a39da007f)